client/tw/categorical.ts

```diff
@@ -139,8 +123,8 @@ export class CatValues extends CategoricalBase {
 
 		// GDC or other dataset may allow missing term.values
 		if (!term.values) term.values = {}
-		const numVals = Object.keys(tw.term.values).length
-		// GDC or other dataset may allow empty term.values
+		//const numVals = Object.keys(tw.term.values).length
+		//GDC or other dataset may allow empty term.values
 		//if (!numVals) throw `empty term.values`
 		if (q.mode == 'binary') {
```

Comment:
Should not allow `mode='binary'` here. Should only be allowed for custom groupsetting (for categorical term).

When verifying `mode='binary'`, should check `tw.q.customset`, not `tw.term.values`.

Can address in a later PR if you prefer.